packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -1465,6 +1465,65 @@ contract SphinxUtils is SphinxConstants, StdUtils {
         return gasEstimates;
     }
 
+    /**
+     * Handles adding an execute action that confirms the Safe has received the requested funding from
+     * our backend. We only include this check if the user requests funds from our backend. This check
+     * just protects the user from an error occurring in our backend which causes the funds to fail to
```

Comment:
How likely is it that our backend will accidentally transfer funds to the user's Gnosis Safe more than once? I remember a note in the previous PR that mentions it's possible if we call `attemptDeployment` more than once

Reply:
Very unlikely, I've already solved this case in the executor.